paasta_tools/utils.py

```diff
@@ -857,6 +857,39 @@ def get_cluster_fqdn_format(self):
         :returns: A format string for constructing the FQDN of the masters in a given cluster."""
         return self.get('cluster_fqdn_format', 'paasta-{cluster:s}.yelp')
 
+    def get_chronos_config(self):
+        """Get the chronos config
+
+        :returns: The chronos config dictionary"""
+        try:
+            return self['chronos_config']
+        except KeyError:
+            raise PaastaNotConfiguredError(
+                'Could not find chronos_config in configuration directory: %s' % self.directory
+            )
+
+    def get_performance_check_config(self):
+        """Get the performance check config
+
+        :returns: The performance_check config dictionary"""
+        try:
+            return self['performance_check']
+        except KeyError:
```

Comment:
I don't think it' is that crazy for these things to instead return "defaults", which may be just an empty dict, and when the code that needs to load a key and really needs it, let *it* complain?